README.rst

```diff
@@ -70,9 +70,111 @@ We appreciate any help!
 BUILD & DEVELOPMENT
 -------------------
 
-Run ``python3 setup.py install``
+1. Installation instructions for **Window** users:
 
-Note: You need to have bower installed before you may install the python packages.
+   - Clone the **coala-html** repository
+
+   ::
+
+    $ git clone https://github.com/coala/coala-html.git
+
+   - Download and install the latest version of Node.js from `official website <https://nodejs.org/en/download/>`__.
+
+   - Change the directory to **coala-html** and run following command to install dependencies
```

Comment:
Maybe add the `cd` command too? Without it is also fine.

Reply:
if added `cd` then it will become a terminal execution instruction, which won't fit in the statement. Without `cd` looks fine :+1: